L1Trigger/L1THGCal/interface/HGCalBackendDetId.h

```diff
@@ -58,8 +65,11 @@ class HGCalBackendDetId : public DetId {
   static const int kHGCalTypeMask = 0x7;
   static const int kHGCalZsideOffset = 16;
   static const int kHGCalZsideMask = 0x1;
+  static const int kHGCalClassIdentifierOffset = 24;
+  static const int kHGCalClassIdentifierMask = 0x1;
 
   enum BackendType { LpGBT, Stage1FPGA, Stage1Link, Stage2FPGA };
+  enum HGCalClassIdentifier { ModuleDetId, BackendDetId };
```

Comment:
This is common to `HGCalBackendDetId` and `HGCalModuleDetId` so the enum should be defined in an external header file. The enum should not be duplicated in the two detid classes.

Reply:
I've added a new mini-class `HGCalBackendCommon` (not sure if this is the best way to do it)

Reply:
A class is not needed. The enum can simply be defined as in `ForwardSubdetector.h`:
 https://github.com/PFCal-dev/cmssw/blob/hgc-tpg-devel-CMSSW_11_2_0_pre5/DataFormats/ForwardDetId/interface/ForwardSubdetector.h

Reply:
thanks - I've modified it